package.json

```
@@ -77,7 +80,7 @@
     "test:update": "jest --updateSnapshot",
     "test:watch": "jest --watch",
     "test:inspect": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand",
-    "lint": "eslint --ignore-path .gitignore --ext .js,.cjs,.mjs,.ts,.tsx .",
+    "lint": "eslint --ignore-path .gitignore --ext .js,.cjs,t.mjs,.ts,.tsx .",
```

Looks like a mistake. We should revert to avoid merge conflicts.
```suggestion
    "lint": "eslint --ignore-path .gitignore --ext .js,.cjs,.mjs,.ts,.tsx .",
```